src/TeamData.elm

```diff
@@ -168,3 +167,12 @@ update msg model =
 subscriptions : Sub Msg
 subscriptions =
     Sub.none
+
+
+nameCheck : Model -> Bool
+nameCheck model =
+    if model.scouterName == "" then
+        False
+
+    else
+        True
```

Comment:
The `if`'s condition is a value just like `False` or `True`, so this can be shortened to any of the following:
```elm
checkName : Model -> Bool
checkName model =
    not (model.scouterName == "")
```
or
```elm
checkName : Model -> Bool
checkName { scouterName } =
    scouterName /= ""
```
or
```elm
checkName : Model -> Bool
checkName =
    ((/=) "") << .scouterName
```
or
```elm
checkName : Model -> Bool
checkName =
    not << String.isEmpty << .scouterName
```
Also, we prefer function names to be verbs rather than nouns - so `checkName` is better than `nameCheck`. It usually increases readability.